lib/unicorn_wrangler/rss_reader.rb

```diff
@@ -0,0 +1,50 @@
+module UnicornWrangler
+  class RssReader
+    LINUX = RbConfig::CONFIG['host_os'].start_with?('linux')
+    PS_CMD = 'ps -o rss= -p %d'.freeze
+    UNITS  = {
+      b:  1024**0,
+      kb: 1024**1,
+      mb: 1024**2,
+      gb: 1024**3,
+      tb: 1024**4,
+    }.freeze
+
+    def initialize(logger:)
+      @logger = logger
+    end
+
+    # Returns RSS in bytes; should work on Linux and Mac OS X
+    def rss(pid: Process.pid)
```

Comment:
this is really meant to the primary entrypoint, I just added the `rss_mb` because I was lazy :)  I suppose I could just make `rss` return mb since we're the only consumer.

Reply:
yep!